compliance/sparql/src/test/java/org/eclipse/rdf4j/sail/memory/MemorySPARQL11QueryTest.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.sail.memory;
+
+import junit.framework.Test;
+import org.eclipse.rdf4j.query.Dataset;
+import org.eclipse.rdf4j.query.parser.sparql.manifest.SPARQL11ManifestTest;
+import org.eclipse.rdf4j.query.parser.sparql.manifest.SPARQLQueryTest;
+import org.eclipse.rdf4j.repository.Repository;
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+
+import java.net.URL;
+
+public class MemorySPARQL11QueryTest extends SPARQLQueryTest {
+
+	public static Test suite() throws Exception {
+
+		URL manifestUrl = SPARQL11ManifestTest.class.getResource("/testcases-sparql-1.1/manifest-evaluation.ttl");
+
```

Comment:
@jeenbroekstra I'm wondering about these tests suites.....couldn't find that they were being run anywhere, created my own classes to run them, quite a few of them are failing.....do we run these somewhere else? Or have we abandoned them?

Reply:
I'm not sure I understand. This is a compliance test class, if you run `mvn verify` it gets executed, so these tests are very much part of our suite and are run every time the CI system builds/tests the project.

Reply:
We have not. They are executed by the classes in package `org.eclipse.rdf4j.query.parser.sparql.manifest`, part of the sparql-compliance module.

Reply:
Ah, hang on, I see what you mean, you're talking specifically about the `testcases-sparql-1.1` directory, as opposed to `testcases-sparql-1.1-w3c`.

Hm, yes, these look abandoned. From the top of my head these were tests I used while SPARQL was in working draft stage (so before there was a complete official W3C test suite), and I think we initially tried to use it to capture our own additional test cases, but it turned out to be difficult to maintain in parallel with the official W3C suite (and using manifests is cumbersome for running individual tests, as you know). So we switched to using only the official manifest test suite, and for our own additional test cases just adding test cases to the ComplexSPARQLQueryTest class.

Reply:
I would not worry about these for now - this needs a cleanup and perhaps there are tests in there that we shouldn't abandon, but it's also likely that quite a few of these tests are based on a draft stage of SPARQL, where the later behavior in the standard changed (as a case in point a lot of the property path cases cover things like `ex:prop{1,2}` which was a draft proposal at one point, but never made it into the standard).